py3status/modules/imap.py

```diff
@@ -42,13 +43,19 @@ def check_mail(self, i3s_output_list, i3s_config):
 
     def _get_mail_count(self):
         try:
-            connection = imaplib.IMAP4_SSL(self.imap_server, self.port)
-            connection.login(self.user, self.password)
-            connection.select(self.mailbox)
-            unseen_response = connection.search(None, self.criterion)
-            mails = unseen_response[1][0].split()
-            mail_count = len(mails)
+            mail_count = 0
+            directories = self.mailbox.split(',')
+            
+            for directory in directories:
+                connection = imaplib.IMAP4_SSL(self.imap_server, self.port)
+                connection.login(self.user, self.password)
```

Comment:
Maybe we could only connect once and login once before the loop (and close after) and instead only select the given directory in the loop ?